java/google/registry/rde/imports/XjcToHostResourceConverter.java

```diff
@@ -60,6 +69,23 @@ static HostResource convert(XjcRdeHost host) {
         .build();
   }
 
+  private static HistoryEntry createHistoryEntry(XjcRdeHost host) {
+    XjcRdeHostElement element = new XjcRdeHostElement(host);
```

Comment:
Fixed - except in XjcToDomainResourceConverter, where this would make the convert method far more verbose due to the extra objects.